### ISSUE TITLE
The permissions are not correctly added to the Info.plist

### ISSUE THREAD

**Issue by ignaciolarranaga** [NativeScript/nativescript-camera#16]
I don't know why, but the permissions requested are not automatically added to the application generated. I mean, the plugins infrastructure is supposed to add the permissions requested by the plugin.

I'm using
- tns-core-modules@2.5.0-2016-10-21-4514
- tns-core-modules-widgets@2.5.0-2016-10-20-133 

Steps to reproduce in the TestCamera app:
1) Remove everything (rm -rf node_modules/ lib/ hooks/ platforms/)
2) npm i
3) tns prepare ios
4) Check out the permissions in XCode and they will not be there:

![screen shot 2016-10-22 at 08 40 57](https://cloud.githubusercontent.com/assets/15932965/19619060/66bcc054-9833-11e6-8221-7dfff55613d2.png)

By the way, if you do the same steps but with the permissions in the Info.plist of the CameraTest (app/App_Resources/iOS/Info.plist) they do appear:

```
    <key>NSPhotoLibraryUsageDescription</key>
    <string>Requires access to photo library.</string>
    <key>NSCameraUsageDescription</key>
    <string>Requires access to camera.</string>
```

![screen shot 2016-10-22 at 08 45 55](https://cloud.githubusercontent.com/assets/15932965/19619087/587a57d0-9834-11e6-8af7-1b5f60a6e575.png)

**Comment by ignaciolarranaga** [comment]
Hi @enchev just in case, is something related to this also happening at other plugins ?, seems not to be a problem of this plugin specifically.

**Comment by nsndeck** [comment]
Fixed with commit #20. The problem was that creating plugin from included scripts on mac this results in a wrong directory structure `plugin/ios/Info.plist` instead `plugin/platforms/ios/Info.plist`.

Also will update nativescript-cli in order to print more info about info.plist merging process.
